assets/src/application/ui/layout/espressoTable/style.scss

```diff
@@ -1,7 +1,6 @@
 .ee-rspnsv-table-wrapper {
 	margin: 0 auto var(--ee-margin-smaller);
 	max-width: 100%;
-	overflow-x: auto;
```

Comment:
![Peek 2020-05-15 14-36](https://user-images.githubusercontent.com/1477453/82046457-95872680-96b9-11ea-957b-934db5c47652.gif)

Reply:
@tn3rb is there any functionality which depends on that `overflow-x: auto;` ?

Reply:
found a relate functionality and adapted this property here: https://github.com/eventespresso/event-espresso-core/pull/2839